File: JAVASCRIPT/Questões uri/1071.js
let valor=+lines.shift()
let valor1=+lines.shift()
let maximo=0
let minimo=0
let calc=0

if (valor>valor1){
    maximo=valor
    minimo=valor1
}else{
    maximo=valor1
    minimo=valor
}
for(let cont=minimo+1;cont<maximo;cont++){
    if((cont%2)!=0){
        calc+=cont
    }
}
console.log(calc)

File: PYTHON/Questões Uri/1071.py
valor=int(input())
valor1=int(input())
minimo=0
maximo=0
contagem=int(0)
if valor>valor1:
  minimo=valor1
  maximo=valor
else:
  maximo=valor1
  minimo=valor

for cont in range(minimo+1, maximo, 1):
  if(cont%2)!=0:
    contagem+=cont
print(contagem)